visualization/dashboard/src/MLIDashboard/Controls/AxisConfigurationDialog/AxisConfigDialog.tsx

```diff
@@ -178,8 +179,9 @@ export class AxisConfigDialog extends React.PureComponent<IAxisConfigProps, IAxi
                 <PrimaryButton 
                     text={localization.AxisConfigDialog.select}
                     onClick={this.saveState}
-                    className={styles.selectButton}
-                />
+                    className ={styles.selectButton}
+                >Select</PrimaryButton>
```

Comment:
What does adding this do? It should be localized if it does anything, but I'd imagine the text property is all we need?